test/e2e/geojson/geojson2mapml.html

```diff
@@ -0,0 +1,715 @@
+    </mapml-viewer>
+    <mapml-viewer id="expected9" projection="OSMTILE" zoom="10" lon="-75.7" lat="45.4" controls>
+      <layer- label="Point" checked><map-meta name="extent" content="top-left-longitude=-75.916809, top-left-latitude=45.886964, bottom-right-longitude=-75.516809,bottom-right-latitude=45.26964"></map-meta><map-meta name="projection" content="OSMTILE"></map-meta><map-meta name="cs" content="gcrs"></map-meta><map-feature><map-featurecaption>This is a Point</map-featurecaption><map-geometry><map-point><map-coordinates>-75.6916809 45.4186964</map-coordinates></map-point></map-geometry><map-properties><h1>String Properties test</h1></map-properties></map-feature></layer->
+    </mapml-viewer>
+    <mapml-viewer id="expected10" projection="OSMTILE" zoom="10" lon="-75.7" lat="45.4" controls>
+      <layer- label="Point 1" checked><map-meta name="extent" content="top-left-longitude=-75.916809, top-left-latitude=45.886964, bottom-right-longitude=-75.516809,bottom-right-latitude=45.26964"></map-meta><map-meta name="projection" content="OSMTILE"></map-meta><map-meta name="cs" content="gcrs"></map-meta><map-feature><map-featurecaption>caption test</map-featurecaption><map-geometry><map-point><map-coordinates>-75.6916809 45.4186964</map-coordinates></map-point></map-geometry><map-properties><h1>This is a Point - with a featurecaption function</h1></map-properties></map-feature></layer->
+    </mapml-viewer>
+    
+
+
+    <!--All the goejson that will be converted to <layer->-->
+      <script>

```

Comment:
A technique [I just learned about today](https://github.com/Maps4HTML/experiments/blob/a1f2ab9d2888f7f43feb66dc1d3786418dab1b3a/api/geojson/canada.js) (TIL), is that you can use `<script src="..."></script>` to load a bunch of geojson-valued objects into the global scope; these can be used in your inline script code and they can be re-used by other tests.